src/components/Table/TableRow/TableRow.tsx

```diff
@@ -1,16 +1,46 @@
-import React, { FC } from "react";
-import { VibeComponentProps } from "../../../types";
+import React, { FC, useCallback, useRef } from "react";
 import styles from "./TableRow.module.scss";
+import { VibeComponentProps } from "../../../types";
 import { ITableCellProps } from "../TableCell/TableCell";
+import { useKeyEvent } from "../../../hooks";
+import { SELECTION_KEYS } from "../../../constants";
+import { NOOP } from "../../../utils/function-utils";
 
 export interface ITableRowProps extends VibeComponentProps {
+  highlight?: boolean;
```

Comment:
1. add documentation to the prop
2. add a story example

Reply:
Done ✅
Don't understand why prop name isn't `selected` or smth... Highlighted is a bit not-straightforward to understand. Let's think twice about the naming before we release this

Reply:
Checkout the story btw, I've done some experimental stuff. Will also consult with Meirav, if you'll like it
![image](https://github.com/mondaycom/monday-ui-react-core/assets/104433616/805e34c6-264d-43e8-87ba-e757666aa956)

Reply:
There will be another feature for "selection" which is adding checkboxes per row, so we didn't want to mix both.
We find `highlight` accurate as it highlights the row but

Reply:
let's remove this tip, it's not relevant

Reply:
So in the future it will be another `selected` prop or what? If row will be selected via checkbox it will be highlighted or not always? Could rows be highlighted, but not selected? 
We are using highlight in `aria-selected` attribute and it's also a bit confusing
What should I write to the description comments?

Btw, maybe the name should be `highlighted` then and not `highlight`?

Reply:
according to the latest discussions, we should not introduce a click functionality on the row.
the only behavior added is to allow a row to become highlighted. the way to trigger it would be by composing some clickable item in a cell.